learn/hoon/hoon-tutorial/egg-timer.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "2.7.1 Walkthrough: Egg Timer"
```

Comment:
Kinda sad at the title not containing "gall" or "app dev" or whatever. You know, for discoverability.

Reply:
It did but it got changed. maybe Gall Walkthrough or Gall Egg Timer or something